packages/server/src/presentation/controllers/logEntriesController.ts

```diff
@@ -51,5 +51,22 @@ logEntriesController.delete(
       }
       res.json();
     }
-  },
+  }
 );
+
+logEntriesController.put('/logs/log-entries', async (req, res) => {
```

Comment:
I didn't follow the pattern above for logId as I feel the id of the logEntry is the unique identifier for updating a single document in the db.